views/includes/page-templates/trust-individual-beneficial-owner.html

```diff
@@ -27,11 +27,27 @@
   <div class="govuk-grid-column-two-thirds">
     {% include "includes/list/errors.html" %}
 
+    {% if formData.relevant_period %}
+      {% set entity_name = formData.entity_name %}
+      {% include "includes/page-templates/important-banner.html" %}
+      {% set pageTitle = pageParams.title + " beneficiary" %}
+    {% endif %}
+
     <span class="govuk-caption-l">{{ trustName }}</span>
     <h1 class="govuk-heading-xl govuk-!-margin-0">{{ pageTitle }}</h1>
-    <br> <p class="govuk-body">You can add more later.</p> <br>
-
-    <form class="form" method="post"> 
+    <br>
+
+    {% if formData.relevant_period %}
```

Comment:
As discussed, we'd like to avoid changing the getPageProperties function and mostly keep closed for modification. However, have refactored to now use pageData instead of formData.

Reply:
Great, sounds good. I like the refactor 👍